yaml/wp_datetime.yaml

```diff
@@ -28,6 +28,7 @@ esphome:
                 timeToSet.month = static_cast<std::uint16_t>(value);
               });
             });
+            CallbackHandler::instance().addCallback(std::make_pair(Kessel,Property::kWOCHENTAG),[](const SimpleVariant&){});
```

Comment:
Unrelated change?

Reply:
yes and no. This one was printing an error message "Callback not found" every time the time was updated. Does not make sense and contributes to spam